src/App.jsx

```diff
@@ -1,16 +1,18 @@
+import React from "react";
 import data from "./data.json";
-//import { Header } from ./Album ?????
-//import { Album } from "./Components/Album";
+import { Header } from "./Components/Header";
+import { extractedAlbums } from "./Components/Album";
```

Comment:
Not sure this one works?
I think it should be enought to import the Album component here (and maybe header), while the rest of the Album´s child components only need to be imported to Album component (probobly depends on the code though)